app/src/main/java/ch/epfl/sweng/SDP/game/drawing/items/AddStarsItem.java

```diff
@@ -0,0 +1,27 @@
+package ch.epfl.sweng.SDP.game.drawing.items;
+
+import ch.epfl.sweng.SDP.auth.Account;
+import ch.epfl.sweng.SDP.game.drawing.PaintView;
+
+public class AddStarsItem extends Item {
+
+    private static final int ADD_STARS = 10;
```

Comment:
I think 10 stars is too big, 3 would be better (at the end of a game you can win 20 stars max).